core-it-suite/src/test/java/org/apache/maven/it/IntegrationTestSuite.java

```diff
@@ -231,6 +231,7 @@ public static Test suite()
         suite.addTestSuite( MavenITmng4679SnapshotUpdateInPluginTest.class );
         suite.addTestSuite( MavenITmng4677DisabledPluginConfigInheritanceTest.class );
         suite.addTestSuite( MavenITmng4666CoreRealmImportTest.class );
+        suite.addTestSuite( MavenITmng4660ResumeFromTest.class );
```

Comment:
In general this is in order of implementation, not in order of reporting (also helps with fast feedback)

Reply:
[Line 723](https://github.com/apache/maven-integration-testing/blob/e8f24ba23d9a8d97a2e1e8c9920213ffb6ebe772/core-it-suite/src/test/java/org/apache/maven/it/IntegrationTestSuite.java#L723) suggests something else...:

>  Add tests in reverse alpha order above.

Reply:
I see, and https://github.com/apache/maven-integration-testing/blob/e8f24ba23d9a8d97a2e1e8c9920213ffb6ebe772/core-it-suite/src/test/java/org/apache/maven/it/IntegrationTestSuite.java#L88-L93 seems contradictionary. This text must be from the period we didn't start with MNG-nnn yet. However, the intention is right: newer tests are more likely to fail, so start with those.
I'll rewrite it...

Reply:
OK, I'll await the outcome of that and then update the place of this new test in the suite.

Reply:
I've put the test at the top of the list. Do you want us to change the javadoc (pointed out by Maarten above) or do you want to do that separately?

Reply:
I will update the javadoc. I've found several places that require an update